tree/dataframe/src/RDFSnapshotHelpers.cxx

```diff
@@ -303,8 +303,6 @@ void SetBranchesHelper(TTree *inputTree, TTree &outputTree, ROOT::Internal::RDF:
 }
 } // namespace
 
-namespace ROOT::Internal::RDF {
```

Comment:
Before this commit, the ROOT build is likely broken, so we would not be able to bisect over this. The function is both qualified and in the namespace, to this commit should be squashed into the commit where the function is moved in its current location, shouldn't it?

Reply:
You can build the repo at any point of the commit history of this branch without breaking :+1: